thirdparty/cmdconfig/commands/cmdsink/cmdsink_test.go

```diff
@@ -136,12 +141,49 @@ spec:
 	}
 }
 
+func TestSinkCommand_Error(t *testing.T) {
```

Comment:
why do we need this in addition to e2e tests?

Reply:
The way these unit tests are written, they are very close to `e2e` tests. I think we can simply replace these with `e2e` tests.

Reply:
These are standalone tests for sink command. It is good to have them to understand sink behavior in isolated fashion. Doesn't harm having them.